test/controllers/application_helper_test.rb

```diff
@@ -21,4 +21,39 @@ def test_add_args_to_url
     assert_equal("/blah\x80",
                  add_args_to_url("/blah\x80", x: "foo\xA0"))
   end
+
+  def test_textile_markup_should_be_escaped
```

Comment:
This is a pre-existing test (see https://github.com/MushroomObserver/mushroom-observer/commit/a38d6cc69e94f4eb7eebc5a852c77f8af4eb37d7) which I moved to tests/**controllers**/application_helper_test.rb from test/**helpers**/application_helper_test.rb.  
The name which I gave to the test is awful.  It implies that that what's being tested is Textile.  But it's really testing MO's escape_html helper method (or maybe the MO `tpl` extension which lives in app/extensions/string_extensions.rb.
I have to think about this.  I will at least find a better name for the test.